bin/env

```diff
@@ -7,3 +7,4 @@ export DOCKER_TAG=${DOCKER_TAG:-"${GIT_BRANCH}-${GIT_HASH}"}
 export DOCKER_TAG=${DOCKER_TAG/\//-}
 
 export DOCKER_IMAGE="${DOCKER_REGISTRY}/debugging-tools:${DOCKER_TAG}"
+export DOCKER_SSH_IMAGE="${DOCKER_REGISTRY}/sshd:${DOCKER_TAG}"
```

Comment:
Wdyt about `debugging-tool-sshd` for clearity that this is primarily used for debugging, I wouldn't want this running on a prod system for too long